src/resolve/resolved.conf.in

```diff
@@ -21,6 +21,12 @@
 # Cloudflare: 1.1.1.1#cloudflare-dns.com 1.0.0.1#cloudflare-dns.com 2606:4700:4700::1111#cloudflare-dns.com 2606:4700:4700::1001#cloudflare-dns.com
 # Google:     8.8.8.8#dns.google 8.8.4.4#dns.google 2001:4860:4860::8888#dns.google 2001:4860:4860::8844#dns.google
 # Quad9:      9.9.9.9#dns.quad9.net 149.112.112.112#dns.quad9.net 2620:fe::fe#dns.quad9.net 2620:fe::9#dns.quad9.net
+#
+# Using DNS= configures global DNS servers and does not suppress link-specific
+# configuration. Parallel requests will be sent to per-link DNS servers
+# configured by NetworkManager(8), systemd-networkd.service(8), or similar
+# management services. Don't touch DNS= unless you understand this. Refer to
+# resolved.conf(5) and systemd-resolved(8).
```

Comment:
i think this is fine to add, but please mention networkd before NM

Reply:
OK, swapped.